dataclients/kubernetes/kubernetestest/api_test.go

```diff
@@ -208,11 +226,14 @@ func TestTestAPI(t *testing.T) {
 	s := httptest.NewServer(a)
 	defer s.Close()
 
-	get := func(uri string, o interface{}) error {
-		return getJSON(s.URL+uri, o)
+	get := func(t *testing.T, uri string, o interface{}) {
+		t.Helper()
+		err := getJSON(s.URL+uri, o)
+		require.NoError(t, err)
```

Comment:
`get` function looks a bit weird as a name if we start to check for error. Maybe `mustGet`